data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2223,7 +2223,7 @@
 
         [message]
             speaker=unit
-            message= _ "Just great, I am cold, wet, and tired and what have we here? Another dead body. This place is starting to get on my nerves!"
+            message= _ "Just great, I am cold, wet, and tired, and what have we here? Another dead body! This place is starting to get on my nerves!"
```

Comment:
Given the speech up to this point in the river, finding another dead body is routine and isn't worthy of a exclamation mark.

I prefer the original for its lack of comma too, it feels like a stream of grumbling.

Reply:
The comma is because I thought was they were to independent clauses joined by a conjunction. The exclamation point wasn't in surprise or alarm. He's just grumbling louder.

Reply:
I also agree with stevecotton, please leave the original.